modules/dicom_archive/php/dicom_archive.class.inc

```diff
@@ -142,6 +143,45 @@ class Dicom_Archive extends \NDB_Menu_Filter
                               );
 
     }
+
+    /**
+     * Override default behaviour, since the page is loaded from mediaIndex.js
+     *
+     * @return void
+     * @access public
+     */
+    function display()
+    {
+        $content = null;
+        $format  = $_REQUEST['format'] ?? '';
+
+        if ($format == 'json') {
+            $content = $this->toJSON();
+        }
+
+        return $content;
+    }
+
+    /**
+     * Create a form to filter media by various criteria
+     *
+     * @return bool
+     * @throws ConfigurationException
+     * @throws DatabaseException
+     * @throws Exception
+     */
+    function _setFilterForm()
+    {
+        $this->addBasicText('patientID', 'Patient ID');
+        $this->addBasicText('patientName', 'Patient Name');
+        $this->addSelect('site', 'Site', \Utility::getSiteList());
+        $this->addBasicText('gender', 'Gender');
```

Comment:
@xlecours 
Gender is better to have a drop-down list.

Reply:
In that specific case, the database field is a varchar so any value could be put there.